cargo-apk/README.md

```diff
@@ -26,81 +26,128 @@ cargo install --path .
 Following configuration options are supported by `cargo apk` under `[package.metadata.android]`:
 
 ```toml
-# Name of your APK as shown in the app drawer and in the app switcher
-apk_label = "APK Name"
-
-# The target Android API level.
-target_sdk_version = 29
-min_sdk_version = 26
-
-# Virtual path your application's icon for any mipmap level.
-# If not specified, an icon will not be included in the APK.
-icon = "@mipmap/ic_launcher"
-
-# If set to true, makes the app run in full-screen, by adding the following line
-# as an XML attribute to the manifest's <application> tag :
-#     android:theme="@android:style/Theme.DeviceDefault.NoActionBar.Fullscreen
-# Defaults to false.
-fullscreen = false
+[package.metadata.android]
+# Specifies the array of targets to build for.
+build_targets = [ "armv7-linux-androideabi", "aarch64-linux-android", "i686-linux-android", "x86_64-linux-android" ]
 
-# Set the instruction of how the activity should be launched
-# See https://developer.android.com/guide/topics/manifest/activity-element#lmode
-# Defaults to standard.
-launch_mode = "standard"
+# Path to your application's resources folder.
+# If not specified, resources will not be included in the APK.
+resources = "path/to/resources_folder"
 
-# Set the minimum required OpenGL ES version.
-# Defaults to [3, 1]
-opengles_version = [3, 0]
+# Path to the folder containing your application's assets.
+# If not specified, assets will not be included in the APK.
+assets = "path/to/assets_folder"
 
-# Sets the applications screenOrientation.
-# See https://developer.android.com/guide/topics/manifest/activity-element
-# and look for `android:screenOrientation` for possible values
-# Defaults to "unspecified" which makes the system pick an orientation and
-# doesn't give you help with rotation.
-orientation = "sensorLandscape"
+# See https://developer.android.com/guide/topics/manifest/uses-sdk-element
+#
+# Defaults to a `min_sdk_version` of 23 and `target_sdk_version` is based on the ndk's default platform.
+[package.metadata.android.sdk]
+min_sdk_version = 16
+target_sdk_version = 29
+max_sdk_version = 29
 
-# Adds a uses-feature element to the manifest
-# Supported keys: name, required
 # See https://developer.android.com/guide/topics/manifest/uses-feature-element
+# 
+# Note: there can be multiple .feature entries.
 [[package.metadata.android.feature]]
-name = "android.hardware.camera"
+name = "android.hardware.vulkan.level"
+required = true
```

Comment:
Like the old readme this should have `version = "1"` instead of `required = true`, it's a `level` after all?

Reply:
It's a bool based on https://developer.android.com/guide/topics/manifest/uses-feature-element. Should that be changed?

Reply:
Seems like it is a bit of both: https://developer.android.com/guide/topics/manifest/uses-feature-element#vulkan-hw-features

Not sure what this is supposed to be, whether it makes sense to infer `required=true`.